src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -61,7 +61,7 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
     window.location.hostname = redirectUrl;
   }
 
-  useFirestoreConnect({
+  useSparkleFirestoreConnect({
     collection: "venues",
     doc: venueId,
```

Comment:
I am pretty sure this could potentially be `undefined` based on reading it from `useParams`, which would cause a pathologic query that loads all venues.